### PR TITLE
Add stub sponsors page

### PR DESCRIPTION
**Pull request by almarklein**
cc @pya

### CODE REVIEW

**Comment by almarklein** [comment]
mmm, I'll also prepare a dir to put images in

File: pelicanconf.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
              ('About', 'about.html'),  # contact info and who makes up the org and committees
              ('Venue', 'venue.html'),
              ('Program', 'program.html'),
+             #('Sponsors', 'sponsors.html'),
              ('Code of Conduct', 'coc.html'),
```
Comment (almarklein) [inline pelicanconf.py]: uncomment to add link to sponsors page in the menu

File: pelicanconf.py [hunk 2]
```diff
@@ -52,7 +53,7 @@
 
 DEFAULT_PAGINATION = 5
 
-STATIC_PATHS = ['images', 'pdf', 'CNAME']
+STATIC_PATHS = ['static2017', 'images', 'pdf', 'CNAME']
```
Comment (almarklein) [inline pelicanconf.py]: a bit ugly, but I wanted to avoid name clashes with static dirs of other years

**Comment by almarklein** [comment]
This should be ready

**Comment by pya** [comment]
Thanks. Pulled successfully. Will work on it today.

**Comment by pya** [comment]
Added the sponsor description. @pdebuyl can you publish the new page.

**Pull request merged**